com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolumeAuthoring.cs

```diff
@@ -113,6 +113,12 @@ public enum ProbeShadingMode
         [SerializeField]
         private bool m_GreedyDilation = false;
 
+        Dictionary<ProbeReferenceVolume.Cell, MeshGizmo> brickGizmos = new Dictionary<ProbeReferenceVolume.Cell, MeshGizmo>();
+        MeshGizmo cellGizmo;
+
+        // In some cases Unity will magically popuplate this private field with a correct value even though it should not be serialized.
+        // The [NonSerialized] attribute allows to force the asset to be null in case a domain reload happens.
+        [System.NonSerialized]
```

Comment:
I think we landed a fix for this a couple weeks ago (basically just returning the field to null in OnDestroy iirc).
Might be worth keeping this anyway for safety I suppose.

Reply:
Do you have the link of the fix? This issue has been here for years that would be really cool if it was fixed :D

Reply:
I meant, a local fix in the APV code :)

Reply:
https://github.com/Unity-Technologies/Graphics/pull/4035

Reply:
Ah okay